### PR TITLE
feat(fonts_selector): adds additional platforms

### PR DESCRIPTION
**Pull request by yuliroth**
## Description

This PR adds documentation for additional fonts_selectors

## Checklist

* [x] PR is scoped to one task
* [ ] Tests are included
* [x] Documentation is included

* [ ] This PR is not making any UI change
* [ ] This PR is making a UI change
  * [ ] Screenshot(s) included

## QA :

* required test cases:
* specific apps / plugins to test :

### CODE REVIEW

File: book/zappifest/plugins-manifest-format.md [hunk 1]
```diff
@@ -355,6 +355,23 @@ Field Key         | Description
 ###### General Fields:
 - **ios_font_selector**: Will populate fonts dropdown with iOS system fonts and Zapp app custom uploaded fonts
 - **android_font_selector**: Will populate fonts dropdown with Android system fonts and Zapp app custom uploaded fonts
+- **samsung_tv_font_selector**: Will populate fonts dropdown with Samsung TV system fonts and Zapp app custom uploaded fonts
```
Comment (snooky23) [inline book/zappifest/plugins-manifest-format.md]: In my opinion, we need to write here from which version it's supported. this is relevant for all selectors.

File: book/zappifest/plugins-manifest-format.md [hunk 2]
```diff
@@ -355,6 +355,23 @@ Field Key         | Description
...
+- **tvos_font_selector**: Will populate fonts dropdown with TVOS system fonts  and Zapp app custom uploaded fonts
+- **roku_font_selector**: Will populate fonts dropdown with Roku system fonts and Zapp app custom uploaded fonts
+- **lg_tv_font_selector**: Will populate fonts dropdown with LG TV system fonts  and Zapp app custom uploaded fonts
+- **web_font_selector**: Will populate fonts dropdown with WEB system fonts and Zapp app custom uploaded fonts
+Here is the example, in the `custom_configuration_fields` will be shown only the relevat platform font, the rest will be ignored if configured:
+
+```
+"fields": [
```
Comment (snooky23) [inline book/zappifest/plugins-manifest-format.md]: In addition, please add the top-level custom_configuration_fields JSON

Comment (snooky23) [reply book/zappifest/plugins-manifest-format.md]: Like the example below: 
```
"general": {
    "fields": [
        {
            "key": "target_screen_id",
            "label": "Target Screen",
            "type": "screen_selector"
        }
    ]
}
```

File: book/zappifest/plugins-manifest-format.md (hunk 1 above)
Comment (yuliroth) [reply book/zappifest/plugins-manifest-format.md]: its depend on plugin... not the selector, selector sending only a string value

**Pull request merged**